content/topics/solo-learn/sql-intermediate/3-working-with-data/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+content_type: project
+prerequisites:
+  hard:
+  - topics/solo-learn/sql-intermediate/2-table-constraints
+  soft: []
+ready: true
+submission_type: link
+tags:
+- sql
+- solo-learn
+title: Solo Learn - SQL Intermediate - 4 Working with Data
+---
+
+Complete the above-mentioned module on Solo Learn.
+
+N.B. When signing up for the course ensure that you sign up for [SQL Intermediate](https://www.sololearn.com/en/learn/courses/sql-intermediate)
+
+Ensure you check your own profile to ensure that it shows that you have completed the [SQL Intermediate](https://www.sololearn.com/en/learn/courses/sql-intermediate) course.
+
+Please submit the link to your SoloLearn profile, for example, https://www.sololearn.com/profile/21320495
+
+We just need to make sure everyone is keeping up 💚
+
+## Instructions for the reviewer
+
+- Please ensure that the learner submitted a link to their Solo Learn Profile.

```

Comment:
Please only use "Instructios to reviewers" if you need to say something that isn't obvious.

Of course the reviewer sould make sure that the learner handed in a profile. The instructions say that is what they should do. The reviewer should read the instructions.